comp/core/config/go.mod

```diff
@@ -40,10 +41,11 @@ require (
 	github.com/DataDog/datadog-agent/pkg/util/winutil v0.55.0-rc.3
 	github.com/DataDog/viper v1.13.5
 	github.com/stretchr/testify v1.9.0
-	go.uber.org/fx v1.18.2
+	go.uber.org/fx v1.22.0
```

Comment:
I believe we should not update FX version as part of theses changes. Better to update FX on a separate PR 😄

Reply:
agreed

this was an incidental change in https://github.com/DataDog/datadog-agent/pull/26315 and reverted in https://github.com/DataDog/datadog-agent/pull/26719 and https://github.com/DataDog/datadog-agent/pull/26747 is open for upgrading fx

Reply:
fixed in https://github.com/DataDog/datadog-agent/pull/26675/commits/3b905e671d2b25da861f2dce47ecc338c4926b06 after rebase